website/frontend/src/tests/pages/Send/Send.test.tsx

```diff
@@ -0,0 +1,497 @@
+import { decodeHexTypes, createPayloadTypeStructure, Metadata } from '@gear-js/api';
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
+import { screen, render, fireEvent, waitFor } from '@testing-library/react';
+
+import { PROGRAM, PROGRAM_ID, MESSAGE_ID, PROGRAM_WITH_META } from './inputData';
+import { TEST_ACCOUNT, TEST_API } from '../../const';
+import { useProgramMock, useAccountMock, useApiMock } from '../../mocks/hooks';
+
+import * as helpers from 'helpers';
+import * as ApiService from 'services/ApiService';
+import { AlertProvider, ApiProvider, AccountProvider } from 'context';
+import { ProgramModel } from 'types/program';
+import { Send } from 'components/pages/Send/Send';
+import { FormValues } from 'components/pages/Send/children/MessageForm/types';
+import { TypeStructure } from 'components/common/Form/FormPayload/types';
+import { getSubmitPayload, getPayloadValue } from 'components/common/Form/FormPayload/helpers';
+
+type Props = {
+  path: string;
+  initialEntries: string[];
+};
+
+const SendMessagePage = ({ path, initialEntries }: Props) => (
+  <ApiProvider>
+    <AccountProvider>
+      <AlertProvider>
+        <MemoryRouter initialEntries={initialEntries}>
+          <Routes>
+            <Route path={path} element={<Send />} />
+          </Routes>
+        </MemoryRouter>
+      </AlertProvider>
+    </AccountProvider>
+  </ApiProvider>
+);
+
+jest.mock('services/MessagesRequestServices', () => ({
+  messagesService: {
+    fetchMessage: () =>
+      Promise.resolve({
+        result: {
+          source: '0x52970eb8531778ac816303e806c694caf65579dfad5fafa31c2b7b0f61dfd6f2',
+          replyError: 'replyError',
+        },
+      }),
+  },
+}));
+
+describe('send message page tests', () => {
+  const submit = (element: Element) => {
+    fireEvent.submit(element);
+  };
+
+  const changeFieldValue = (element: Element | Node, value: string) => {
+    fireEvent.change(element, { target: { value } });
+  };
+
+  it('should show loader', () => {
```

Comment:
imo it should be part of form tests, to test if loader dissapears after requests are resolved